### DISCUSSION TITLE
DX7s factory performances portamento

### DISCUSSION THREAD

**Discussion by Piattica** [BobanSpasic/MDX_Vault#9] [General]
too much portamento

**Comment by BobanSpasic** [comment]
Both MidiDexed and DX7s have Portamento time in the range 0-99. I am not doing any translations here. What MiniDexed does not have is Portamento Step, but I didn't find any description how this should affect the portamento effect.

**Comment by BobanSpasic** [comment]
Portamento Step means Glissando. If it is 0, then we have a real portamento. Other than 0 means glissando.
So, it has nothing really to do with our problem.
